stripe/src/main/java/com/stripe/android/util/StripeNetworkUtils.java

```diff
@@ -37,13 +41,41 @@
         cardParams.put("address_country", StripeTextUtils.nullIfBlank(card.getAddressCountry()));
 
         // Remove all null values; they cause validation errors
-        for (String key : new HashSet<>(cardParams.keySet())) {
-            if (cardParams.get(key) == null) {
-                cardParams.remove(key);
-            }
-        }
+        removeNullParams(cardParams);
 
         tokenParams.put("card", cardParams);
         return tokenParams;
     }
+
+    @NonNull
+    public static Map<String, Object> hashMapFromBankAccount(@NonNull BankAccount bankAccount) {
+        Map<String, Object> tokenParams = new HashMap<>();
+        Map<String, Object> accountParams = new HashMap<>();
+
+        accountParams.put("country", bankAccount.getCountryCode());
+        accountParams.put("currency", bankAccount.getCurrency());
+        accountParams.put("account_number", bankAccount.getAccountNumber());
+        accountParams.put("routing_number",
+                StripeTextUtils.nullIfBlank(bankAccount.getRoutingNumber()));
+        accountParams.put("account_holder_name",
+                StripeTextUtils.nullIfBlank(bankAccount.getAccountHolderName()));
+        accountParams.put("account_holder_type",
+                StripeTextUtils.nullIfBlank(bankAccount.getAccountHolderType()));
+
+        // Remove all null values; they cause validation errors
+        removeNullParams(accountParams);
+
+        tokenParams.put("bank_account", accountParams);
```

Comment:
Use the `TokenParser. FIELD_BANK_ACCOUNT` constant from TokenParser so that we don't have the String declared in two places?